sesam.py

```diff
@@ -2545,6 +2592,9 @@ def run(self):
 
                         self.token = self.result["token"]
                         while True:
+                            # IS-15613: long-running CI tests are also user interactions
+                            self.sesam_node.register_user_interaction()
```

Comment:
Could you do this call before the while loop to reduce the load on the portal?

Reply:
@grove we already protect against this here: https://github.com/sesam-community/sesam-py/pull/135/files#diff-b00e2cdd790f35273646fa0f925e1d97aa9a384a205a2bfd6fdb3b388e8829f3R258

Reply:
(We do it in the while loop deliberately to avoid hibernating a node that spends too long running a ci test)